lib/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -0,0 +1,11 @@
+.. _liblwm2m_carrier_changelog:
+
+Changelog
+#########
+
+All notable changes to this project are documented in this file.
+
+liblwm2m_carrier 0.6.0
+***************
```

Comment:
I think you need to add some padding here to match the zephyr style

Reply:
Done